cross/busybox/Makefile

```diff
@@ -27,7 +27,7 @@ busybox_configure:
 
 .PHONY: busybox_compile
 busybox_compile:
-	$(RUN) CROSS_COMPILE=$(TC_PATH)$(TC_PREFIX) $(MAKE) busybox
+	$(RUN) CROSS_COMPILE=$(TC_PATH)$(TC_PREFIX) $(MAKE) -j$(NCPUS) busybox
 
 .PHONY: busybox_install
```

Comment:
most of the custom compile-targets adjusted here could be removed by definition of `COMPILE_MAKE_OPTIONS`
as the default compile_target executes `$(RUN) $(MAKE) $(COMPILE_MAKE_OPTIONS)`

busybox_compile can be removed by:
`COMPILE_MAKE_OPTIONS += CROSS_COMPILE=$(TC_PATH)$(TC_PREFIX)  busybox`

this work only, when it does not matter that CROSS_COMPILE definition must not be left of $(MAKE) on the command line.

As spksrc.common.mk already defines `COMPILE_MAKE_OPTIONS += -j$(NCPUS)` this must not be added here again.